### PR TITLE
Adds steps to run multiple pyrsia_nodes as peers on the same computer

### PR DESCRIPTION
**Pull request by betarelease**
Addresses #297

### CODE REVIEW

**Comment by betarelease** [comment]
@jbaruch I just added a few steps that we did not have to run multiple nodes on the same machine. Can you please try these out while reviewing my PR? Thanks.

**Review by jbaruch — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -92,11 +92,35 @@ For now Pyrsia only supports Docker artifacts. Follow these steps to run a Pyrsi
...
+
+    ```
+    ./pyrsia node -s
+    Connected Peers Count:   1 # Shows the additional node that joined the list of peers
     Artifacts Count:         12
     Total Disk Available:    983112
     ```
 
+    In a real life deployment these nodes will be spread over the network and will all run on their own 7888 port.
+    `Word of caution: Running the peers for a few hours does generate network traffic and hence can drain your computer power. Ensure you are plugged into power if you are running multiple peers for a long time`
```
Comment (mgrand) [inline readme.md:121]: This probably should not be formatted at code. Perhaps
```suggestion
    <br>**Word of caution:** Running the peers for a few hours does generate network traffic and hence can drain your computer power. Ensure you are plugged into power if you are running multiple peers for a long time.
```

File: readme.md [hunk 2]
```diff
@@ -109,5 +133,5 @@ For now Pyrsia only supports Docker artifacts. Follow these steps to run a Pyrsi
     * macOS and Windows: Compose is included in Docker Desktop
     * Linux: [Downloaded Compose](https://github.com/docker/compose#linux)
 
-The Pyrsia node will then be running on localhost:7888 both on the host and 
+The Pyrsia node will then be running on localhost:7888 both on the host and
```
Comment (mgrand) [inline readme.md:136]:
```suggestion
The Pyrsia node will then be listening on localhost:7888 both on the host and
```

**Review by mgrand — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (betarelease) [reply readme.md:121]: Missed this comment. I meant to highlight it in some way and using single ticks was the easiest. I prefer to not use html when writing markdown so chose the simplest way.

**Pull request merged**